src/monattack.cpp

```diff
@@ -113,6 +113,9 @@ void mattack::antqueen(monster *z, int index)
 
 void mattack::shriek(monster *z, int index)
 {
+    if( z->friendly ) {
+        return; // TODO: handle friendly monsters
```

Comment:
Friendly shrieker might shriek with good intentions, but would only summon more hostiles.  Either shrieking or not-shrieking is acceptable, but I'd rule shrieking as it's not smart enough to know that it shouldn't.